### ISSUE TITLE
[IDEA] square bracket field transclusion

### ISSUE THREAD

**Issue by justin-hurd** [Jermolene/TiddlyWiki5#6780]
I'm not sure how to word it, but the ability to link to a tiddler while using the value of a specific field, ie.

[[GettingStarted!!caption]] or [[GettingStarted!!alias]]

Alternatives to this is [[Hello|GettingStarted]] but in doing this, the user has to manually decide what they what to name the link, which could be mispelled.

**Comment by justin-hurd** [comment]
@twMat Not quite, the result for both of those is a missing link to hello, rather than a link to GettingStarted.

in plain HTML with Wikitext it would look something like, "<a href="#GettingStarted">{{GettingStarted!!caption}}</a>" though this only works the first time you click the link, I imagine I did it partially incorrect but it should work the first time to get an example.

The reason I suggest the shortened wikitext of [[Tiddler!!Field]] is incase you're typing about someone and don't remember their name spelling, and have it abbriviated, or maybe they have special characters in their name, etc, just as an addendum to my previous reasoning.

**Comment by kookma** [comment]
A linkify macro has been discussed in forum and also is part of Utility plugin.

See the small macro here: https://talk.tiddlywiki.org/t/link-to-tiddler-by-any-field/588

**Comment by pmario** [comment]
@justin-hurd ... The main problem with `[[alias|?c]]` links is the `backlinks[]` filter operator. So if you have TW link syntax, you'll also need a performant [indexer for back linking](https://wikilabs.github.io/editions/uni-link/).

Since there are several plugins that can handle it, I'll close as: not planned.
